services/src/api/handlers/wfs.rs

```diff
@@ -499,14 +498,15 @@ async fn wfs_feature_handler<C: ApplicationContext>(
             sources: SingleRasterOrVectorSource {
                 source: RasterOrVectorOperator::Vector(operator),
             },
-        };
+        }
+        .boxed();
+
+        let workflow_operator_path_root = WorkflowOperatorPath::initialize_root();
 
-        // create the inititalized operator directly, to avoid re-initializing everything
-        let ivp = InitializedVectorReprojection::try_new_with_input(
-            CanonicOperatorName::from(&reprojected_workflow),
-            reprojection_params,
-            initialized,
-        )?;
+        // TODO: avoid re-initialization and re-use unprojected workflow. However, this requires updating all operator paths
```

Comment:
I don't get it.

Reply:
in order to check whether we need to inject a reprojection, we first need to initialize the original workflow. Then we can check the result projection. Previously, we then just wrapped the initialized workflow with an initialized reprojection. IMHO this is wrong because initialization propagates the workflowpath down the children and append a new segment for each level. so we can't re-use an already initialized workflow.

Reply:
Sollte nicht nur der Elternknoten länger werden? Die source würde ja Stufe 1 bleiben, aber oben kommt etwas dazu.